doc/source/ray-observability/state/state-api.rst

```diff
@@ -422,14 +422,14 @@ Logs
 State API also allows you to conveniently access ray logs. Note that you cannot access the logs from a dead node.
 By default, the API prints log from a head node.
 
-E.g., Get all retrievable log file names from a head node
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+E.g., Get all retrievable log file names from a head node in a cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. tabbed:: CLI
 
     .. code-block:: bash
 
-        ray logs 
+        ray logs cluster
```

Comment:
Btw it is confusing from the doc that only the first example is using `ray logs cluster` and other examples start using `ray logs <file>`. We should fix this as a follow up (use a consistent example and put a note ray logs is a alias of ray logs cluster)